UpBeatRadio/dist/metadata.json

```diff
@@ -1,7 +1,13 @@
 {
   "author": {
-    "name": "thatdevjon",
-    "id": "570656255610191874"
+    "name": [
+      "thatdevjon",
+      "Dominik"
+    ],
+"id": [
+  "570656255610191874",
+  "594491170248458250"
+]
```

Comment:
This is wrong, you can only have 1 author. You can add yourself as a contributor, please look at the docs for more info on how to add it.